app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/Router.scala

```diff
@@ -19,12 +19,21 @@ package uk.gov.hmrc.cdsreimbursementclaimfrontend.controllers
 import play.api.mvc.Call
```

Comment:
(Not to be addressed on this PR): I'm increasingly concerned by calling this class "Router" and the journeys "Routes" as that overloads the terminology that Play has for Router/Routes. If the code stays beyond the review going on this week it would be good to consider renaming it. eg "Journey"

Reply:
Ok